packages/pluginutils/types/index.d.ts

```diff
@@ -18,7 +18,7 @@ export interface DataToEsmOptions {
 }
 
 /**
- * A valid `minimatch` pattern, or array of patterns.
+ * A valid `picomatch` pattern, or array of patterns.
```

Comment:
We should mention glob somewhere too
```suggestion
 * A valid `picomatch` glob pattern, or array of patterns.
```